untitled/Study/Decorator_Property.py

```diff
@@ -0,0 +1,47 @@
+class Triangle:
+    def __init__(self, a, b, c):
+        self.__a = a
+        self.__b = b
+        self.__c = c
+
+    @property
+    def property_perimeter(self):
+        self.perimeter = self.__a + self.__b + self.__c
+        return self.perimeter
```

Comment:
No need to store the return value in the instance variable. We can just return the sum of private attributes. This is the main purpose of computed properties. 
```suggestion
        return self.__a + self.__b + self.__c
```
With the comment above and you will get a computed `perimeter` property that can be got by `t.perimeter`.